cpp/velox/compute/VeloxPlanConverter.h

```diff
@@ -35,45 +35,19 @@ class VeloxPlanConverter {
       const std::unordered_map<std::string, std::string>& confMap,
       bool validationMode = false);
 
-  std::shared_ptr<const facebook::velox::core::PlanNode> toVeloxPlan(::substrait::Plan& substraitPlan);
+  std::shared_ptr<const facebook::velox::core::PlanNode> toVeloxPlan(
+      ::substrait::Plan& substraitPlan,
```

Comment:
Not your change. Maybe better to add const here.

Reply:
ok.